docker-compose.yml

```diff
@@ -87,31 +88,19 @@ services:
     environment:
       - DB_NAME=signals
       - DB_PASSWORD=insecure
-      - UWSGI_HTTP=0.0.0.0:8000
-      - UWSGI_MASTER=1
-      - UWSGI_STATIC_INDEX=index.html
-      - UWSGI_STATIC_MAP=/signals/static=/static
-      - UWSGI_CALLABLE=application
-      - UWSGI_VACUUM=1
-      - UWSGI_STATIC_EXPIRES=/* 3600
-      - UWSGI_OFFLOAD_THREADS=1
-      - UWSGI_HARAKIRI=15
-      - UWSGI_DIE_ON_TERM=1
-      - UWSGI_MODULE=signals.wsgi:application
       - DJANGO_SETTINGS_MODULE=signals.settings.development
+      - SWIFT_ENABLED=False
       - SIGMAX_AUTH_TOKEN
       - SIGMAX_SERVER
       - DWH_MEDIA_ROOT=/dwh_media
       - ELASTICSEARCH_HOST=elasticsearch:9200
-      - ELASTICSEARCH_INDEX=signals
-      - SWIFT_ENABLED=False
     volumes:
       - ./api/app:/app
       - ./api/deploy:/deploy
       - ./dwh_media:/dwh_media
-    command: bash -c "
-      python /app/manage.py collectstatic --noinput &&
-      uwsgi --py-auto-reload=1"
+      - ./scripts/initialize.sh:/initialize.sh
+    command:
+      - /initialize.sh
```

Comment:
We often restore dumped databases locally. For that usecase it makes no sense to initialize the database with dummy data. I would like to just start the application without data and then run scripts to bootstrap or restore a database dump.

Reply:
At this moment it is possible to start the api without bootstrapping it. It is a bit overcomplicated, if the initialize log file exists no boostrapping is done and the api is started. Maybe we can add an ENV variable which can be set to skip bootstrapping?